backend/server/src/main/java/ai/verta/modeldb/utils/RdbmsUtils.java

```diff
@@ -51,63 +48,6 @@ public class RdbmsUtils {
       "' and entity_name = 'ExperimentRunEntity') o, (select id from keyvalue where kv_key ='";
   private static final String MAX_EPOCH_NUMBER_SQL_3 =
       "' and  entity_name IS NULL) k where o.keyvaluemapping_id = k.id ";
-  private static final String AND_QUERY_OPERATOR = " AND ";
```

Comment:
this class had a lot of unused stuff in it, and I ran into it while tracking down the NPEs, so I cleaned it up